CHANGES.md

```diff
@@ -1,3 +1,28 @@
+## edge-19.6.3
+
+* CLI
+  * Added a check for Pod Security Policy (PSP) permissions to `linkerd check`
+  * Changed `linkerd check` to only warn (not error) if PSPs are found, but
+    the caller does not have the `NET_ADMIN` capability
+* Controller
+  * Improved timestamp formatting in logs
+  * Default the mutating and validating webhook configurations `sideEffects` 
+    property to `None` to indicate that the webhooks have no side effects on
+    external resources
+* Proxy
+  * Added the `NET_RAW` capability to the proxy-init container to prevent it
+    from failing when the primary container has certain security contexts
+  * Fixed the handling of HTTP2 requests that don't have an `:authority`
+  * Introduced eager eviction to the router cache so that idle services are
```

Comment:
This seems like it could be made a little less wordy somehow? What about something like 
> Changed the proxy to eagerly evict idle services from router caches, even when they are not full